torch/csrc/autograd/python_variable.cpp

```diff
@@ -522,6 +536,8 @@ static struct PyGetSetDef THPVariable_properties[] = {
   {"layout", (getter)THPVariable_layout, nullptr, nullptr, nullptr},
   {"device", (getter)THPVariable_device, nullptr, nullptr, nullptr},
   {"ndim", (getter)THPVariable_get_ndim, nullptr, nullptr, nullptr},
+  {"nbytes", (getter)THPVariable_nbytes, nullptr, nullptr, nullptr},
```

Comment:
can you add 'nbytes' and 'itemsize' to this list: https://github.com/pytorch/pytorch/blob/b37c235d8647b5e4be53ee482cc50e244fe3f1ac/tools/autograd/gen_python_functions.py#L19-L40

just so we don't accidentally try to bind the methods in python in the future.